plugin-server/src/worker/ingestion/hooks.ts

```diff
@@ -362,7 +362,10 @@ export class HookCommander {
                 `⌛⌛⌛ Posting Webhook slow. Timeout warning after 5 sec! url=${webhookUrl} team_id=${team.id} event_id=${event.eventUuid}`
             )
         }, 5000)
-        const relevantFetch = isCloud() && this.fetchHostnameGuardTeams.has(team.id) ? safeTrackedFetch : trackedFetch
+        const relevantFetch =
+            isCloud() && (!this.fetchHostnameGuardTeams || this.fetchHostnameGuardTeams.has(team.id))
```

Comment:
nit: I'd re-add the comment here about empty = enabled for everyone as that's hard to reason about otherwise